cargo-verify/src/main.rs

```diff
@@ -395,6 +404,14 @@ fn verify(opt: &Opt, package: &str, target: &str) -> CVResult<Status> {
     info_at!(&opt, Verbosity::Informative, "  Building {} for verification", package);
     let bcfile = build(&opt, &package, &target)?;
 
+    info_at!(&opt, Verbosity::Informative, "  Generated binary {}", bcfile.to_string_lossy());
+
+    if let Some(output) = &opt.output {
+        std::fs::copy(bcfile, output)?;
+        info_at!(&opt, Verbosity::Informative, "Wrote binary to {}", output.to_string_lossy());
+        return Ok(Status::Verified)
```

Comment:
Maybe use `exit(0)` instead of `return`? This will avoid printing the "VERIFICATION_RESULT: {}" which is misleading.

Reply:
Good idea - done